### ISSUE TITLE
Base64 encode fields in Secret

### ISSUE THREAD

**Issue by vroldanbet** [aptakube/aptakube#87]
### What problem are you trying to solve? Do you a suggestion on how to solve it?

Secret fields are base64 encoded. Whenever I need to edit a secret manually it involves using an external tool to base64 encode the values. Automatically base64 encoding the values for the user would accelerate the development lifecycle.

For example, Lens does this by allowing the user to modify the individual values and base64 encoding them on save.

From https://kubernetes.io/docs/concepts/configuration/secret/

>You can specify the data and/or the stringData field when creating a configuration file for a Secret. The data and the stringData fields are optional. The values for all keys in the data field have to be base64-encoded strings. If the conversion to base64 string is not desirable, you can choose to specify the stringData field instead, which accepts arbitrary strings as values.

**Comment by mschneider82** [comment]
+1 @goenning  after solving this, a user can uninstall lens. Editing encoded secrets or multiline configmaps could be improved.
Maybe allow edit in the Overview data preview?

**Comment by marekjelen** [comment]
agreed, secrets management is pretty much the only reason I still have Lens installed

Any chance this could be bumped a little bit? @goenning

**Comment by goenning** [comment]
note: it's already possible using the `stringData` without having to encode it.

but let me see how much work it's to add a better UI for it

**Comment by marekjelen** [comment]
true, in that case I would probably go with  `base64` encoding the string before pasting, this always felt too *magicy* for such an important resource

though, having UI for editing these kind of resources (secrets, configMaps) would be convenient and with e.g. config files less error prone when dealing with multiline values

**Comment by mschneider82** [comment]
this feature request would be a huge benitfit. I often deal with secrets, rather then needing a UI for helm things.

A simple "Selection base64 decode / encode" in the yaml editor would be fine for me.

**Comment by goenning** [comment]
Something like this @marekjelen @mschneider82 ?

https://github.com/user-attachments/assets/0125ebbb-962b-453b-8941-d20a014765a6

**Comment by marekjelen** [comment]
@goenning yes, sire, that looks pretty much like what I was looking for 👍

**Comment by goenning** [comment]
This was released on 1.10 today. Give it a try and let me know what you like/dislike.

**Comment by marekjelen** [comment]
Any secret I am trying to edit, I am getting this error.

<img width="425" alt="Image" src="https://github.com/user-attachments/assets/a3d94e59-e3a5-4023-898c-f8c154ffa080" />

**Comment by goenning** [comment]
Are you on version 1.10.2 or higher?

**Comment by goenning** [comment]
And if yes, could you share the secret value (omit any actual sensitive value , this seems like a format / special character issue)

**Comment by marekjelen** [comment]
I am sorry, my bad, I was on version 

```
Version 1.10.0 (20250115.201721)
```

The upgrade fixed the problem.

Thanks!! You and Aptakube are awesome! 🎆
